<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompetitionInfoEditorWidget</class>
 <widget class="QWidget" name="CompetitionInfoEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,10">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_hill">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Quicksand Medium</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Skocznia</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_hill">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 6px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_hill_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Quicksand Medium</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background: transparent;
color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Rodzaj serii</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_seriesType">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: rgb(222, 221, 218);
	color: rgb(60, 60, 60);
	border: 2px solid rgb(65, 0, 0);
	border-radius: 6px;
}

QDoubleSpinBox::up-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
		background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}
QDoubleSpinBox::down-button{
border: 1px solid rgb(33, 42, 0);
border-radius: 3px;
	background-color: rgb(129, 175, 117);
	color: rgb(0, 59, 23);
}</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <item>
        <property name="text">
         <string>Konkurs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kwalifikacje</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Seria pr√≥bna</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Trening</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_competitionRules"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_classifications"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
